test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3117,27 +3117,66 @@ def test_groupnorm_noaffine(self):
 
     @skipIfUnsupportedMinOpsetVersion(9)
     def test_listunpack(self):
```

Comment:
can you please rename this to `test_list_unpack`? Generally `CamelCase` -> `snake_case`, not `snakecase`

Reply:
There is another `test_list_unpack` which was testing the non scripted version of the model. Is there a convention we use to distinguish scripted and non-scripted tests?

Reply:
Hah! Not that I know of, and if that is a convention we should not continue it. Normally one test covers both. If something can only run in script mode sometimes it is named with `script`, which seems like a much better convention to me.

Reply:
Done